classes/global/touch_control/touch_button.gd

```diff
@@ -12,9 +22,11 @@ func _ready():
 
 
 func _setup_textures(new_id: String):
-	var ypos = ID_LIST.find(new_id) * 21
-	normal.region.position.y = ypos
-	pressed.region.position.y = ypos
+	var index = ID_LIST.find(new_id)
+	var pos = Vector2(floor(index / 7), index % 7)
+	pos *= Vector2(40, 21)
```

Comment:
Some comments explaining this math would be very appreciated

Reply:
+could the 7 be moved to a variable rather than sitting there as a magic number?

Reply:
I must admit I rushed this one a bit. Will fix soon.